src/v2i-hub/MUSTSensorDriverPlugin/scripts/MockMUSTSensor.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/python3
+
+import socket
+import sys
+import time
+import argparse
+from dataclasses import dataclass
+from enum import Enum
+
+
+class DetectionClassification(Enum):
+    """Enumeration used for indentifying type of detection
+    """
+    SEDAN='sedan'
+    VAN='van'
+    TRUCK='truck'
+class DetectionSize(Enum):
+    """Enumeration used for indentifying the type of KafkaLogMessage
+    """
+    SMALL='small'
+    MEDIUM='medium'
+    LARGE='large'
+
+@dataclass
+class MUSTDetection:
+    """Class used to store data for each Kafka Log Message
+    """
+    classification: DetectionClassification
+    x: float
+    y: float
+    heading: float
+    speed: float
+    size: DetectionSize
+    confidence: float
+    track_id: int
+    timestamp: int
+
+    def to_csv(self):
+        return f'{self.classification.value},{self.x},{self.y},{self.heading},{self.speed},{self.size.value},{self.confidence},{self.track_id},{self.timestamp}'
+
+def move_detection():
+    return 
+
+def create_socket():       
+    try:
+        return socket.socket(family=socket.AF_INET, type=socket.SOCK_DGRAM)
+    except socket.error as err:
+        print('Socket error because of %s' %(err))
```

Comment:
The same comments as below. Also, what possible case this error is thrown?

Reply:
Providing an invalid host would trigger this error. No retry would fix that.